From 2620295542d1b3867875056c8d0ca9fbcd0546a0 Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <d.kolesa@samsung.com>
Date: Thu, 11 Jun 2020 15:14:03 +0200
Subject: [PATCH 7/7] modules: only build gl_drm evas engine when using es-egl

This module does not work with desktop opengl contexts,
and needs EGL to work.
---
 src/modules/evas/engines/meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git src/modules/evas/engines/meson.build src/modules/evas/engines/meson.build
index 2504a761a9..7bb4211cf2 100644
--- src/modules/evas/engines/meson.build
+++ src/modules/evas/engines/meson.build
@@ -16,10 +16,7 @@ endif
 have_gl_engine = false
 
 if get_option('opengl') != 'none'
-  engines += [
-   ['gl_x11', ['x11']],
-   ['gl_drm', ['drm']]
-  ]
+  engines += [['gl_x11', ['x11']]]
   have_gl_engine = true
 endif
 
@@ -29,7 +26,10 @@ if get_option('opengl') == 'full'
 endif
 
 if get_option('opengl') == 'es-egl'
-  engines += [['wayland_egl', ['wl']]]
+  engines += [
+   ['wayland_egl', ['wl']],
+   ['gl_drm', ['drm']]
+ ]
   have_gl_engine = true
 endif
 
-- 
2.27.0

